west.yml

```diff
@@ -51,7 +51,7 @@ manifest:
       revision: 0ec40aed8087f26bd9ac1b70fb5a6c326a6451aa
       path: modules/hal/st
     - name: hal_stm32
-      revision: 686c93da4186eb9c48b83ee235fc0454693f6e5c
```

Comment:
Split in dedicated commit

Reply:
Missed this comment, this is done thank you !